### PR TITLE
Add pl_programs_offered field to regional partner model

### PR DESCRIPTION
**Pull request by tim-dot-org**
Adds a field to the json properties of the regional partner which indicates which PL courses are planned to be offered by this RP. This field will be used later in the application to notify the teacher whether professional learning for the chosen course is available in their region (spec req 7).

See notes at the bottom of these docs for why blank strings need to be removed from the field:
https://apidock.com/rails/ActionView/Helpers/FormOptionsHelper/select

## Links

- spec: [See req 1](https://docs.google.com/document/d/1rshrcpg7J45BfrgiEl6ALwysDeQiwLQcAErFthocJug/edit#)
- jira ticket: [PLAT-1246](https://codedotorg.atlassian.net/browse/PLAT-1246)

## Testing story

Tested updates manually through regional partner edit form. Added unit test.

image

image

### CODE REVIEW

File: dashboard/app/controllers/regional_partners_controller.rb [hunk 1]
```diff
@@ -64,6 +64,9 @@ def update
       end
     end
 
+    puts 'update'
+    puts update_params
+
```
Comment (davidsbailey) [inline dashboard/app/controllers/regional_partners_controller.rb]: remove debug code?

**Review by davidsbailey — APPROVED** [review]
LGTM!

**Comment by dmcavoy** [comment]
nit but I think pl_programs_offered sounds easier to read. 

Lol NVM. I looked at the code and it is this way. Just the title of the PR is not.

File: dashboard/app/controllers/regional_partners_controller.rb (hunk 1 above)
Comment (tim-dot-org) [reply dashboard/app/controllers/regional_partners_controller.rb]: haha oops

**Pull request merged**